pkg/lint/lint_test.go

```diff
@@ -75,8 +75,13 @@ func TestBadChart(t *testing.T) {
 				e6 = true
 			}
 		}
+		if msg.Severity == support.WarningSev {
+			if strings.Contains(msg.Err.Error(), "version '0.0.0.0' is not a valid SemVerV2") {
+				w = true
+			}
+		}
 	}
-	if !e || !e2 || !e3 || !e4 || !e5 || !i || !e6 {
+	if !e || !e2 || !e3 || !e4 || !e5 || !i || !e6 || !w {
```

Comment:
Table tests are great for somethings, but this is a good example of when they are not great! Comment has nothing to do with what you are doing here though.